LokiLabs/microsoft-graph-devx-api
# AB#39171 Added teamsAppSampleQueries in model and extended ordering functionality

## Overview

Currently the samples endpoint orders the sample queries when returned and this causes new entries to not be returned. This PR fixes that problem by adding teamsAppSampleQueries to the model and the filtering

## Testing Instructions

* Get the repo and the branch running
* Call the endpoint /samples?org=LokiLabs&branchName=ADD_YOUR_BRANCH_NAME_HERE 
* Scroll all the way down to the returned samples and you will find the list teamsAppSampleQueries 

NOTE: ADD_YOUR_BRANCH_NAME_HERE should be a branch based on the devx_content repo, if you dont have a branch or didnt create one. You can use '38584-app-sample-queries'.

ADO: https://dev.azure.com/microsoftgarage/Intern%20GitHub/_sprints/taskboard/GI21%20-%20Graph%20Explorer/Intern%20GitHub/Y21-S/07?workitem=39171
